### PR TITLE
Init firebase

### PR DESCRIPTION
**Pull request by adindotdev**
- If you don't know what this is, ask first! These steps are only for those who are handling CI
- Ask a current developer for access to the Firebase project (via Google account)
- `npm install -g firebase-tools` to install necessary packages for CI
- `firebase login:ci` to login to your Google account. SAVE THE TOKEN. I highly recommend declaring it as an environment variable in `.bashrc` or `.zshrc` depending on your system. If you save it as `FIREBASE_TOKEN`, you won't need include the token with every CI command later.
- `firebase init` in your project root directory. Answer the following prompts as so:
    - Choose the existing **bt-web** project
    - Choose hosting
    - *What do you want to use as your public directory? (public)* **build**
    - *Configure as a single-page app (rewrite all urls to /index.html)?* **N/No**
    - *File build/index.html already exists. Overwrite?* **N/No**
- **!! WARNING !! Should you be doing this?** `firebase deploy` when ready

### CODE REVIEW

**Comment by ianmah** [comment]
Closes #6

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 debug.log
+
+# firebase
```
Comment (ianmah) [inline .gitignore]: Did you add these manually? I think we actually need these and they don't expose anything sensitive

File: public/index.html [hunk 2]
```diff
@@ -39,5 +38,15 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+
+  <!-- The core Firebase JS SDK is always required and must be listed first -->
+  <script src="/__/firebase/7.4.0/firebase-app.js"></script>
```
Comment (ianmah) [inline public/index.html]: When I did this before I didn't need to include any js files, can you link to documentation for this?

Comment (adindotdev) [reply public/index.html]: This isn't necessary for hosting but will be for almost all other fb functionality. I just included it as part of the Firebase project init. The snippet is found in Project Overview -> 1 App -> bt-web Settings. If the other fb features (ie analytics) aren't an immediate interest then we can remove it

File: .gitignore (hunk 1 above)
Comment (adindotdev) [reply .gitignore]: Re-added them back after convo.

**Pull request merged**